MediaManager/Platforms/Apple/AppleMediaManagerBase.cs

```diff
@@ -221,12 +244,14 @@ public override Task SeekTo(TimeSpan position)
 
         public override Task StepBackward()
         {
-            throw new NotImplementedException();
+            var playerTime = NativeMediaPlayer.Player.CurrentTime;
+            return this.SeekTo(TimeSpan.FromSeconds(Double.IsNaN(playerTime.Seconds) ? 0 : ((playerTime.Seconds < 10) ? 0 : playerTime.Seconds - 10)));
```

Comment:
The value of how much should be set in a central place. Can you add a property?

Reply:
Done!